examples/mqtt.eve

```diff
@@ -0,0 +1,21 @@
+
+# MQTT
+
+## Listening to incoming messages
+
+```eve
+search @mqtt
+  m = [#message #incoming topic payload]
+bind @browser
+  [#div text: "MQTT message at {{topic}} : {{payload}}"]
+```
+
+## Sending messages
+
+```eve
+search
+  i = range[from: 1 to: 3]
+
+commit @mqtt
+  [#message #outgoing topic: "eve/test/" + i, payload: i]
```

Comment:
i don't know that we overloaded + in this way? pretty sure the preferred syntax here would
be 'interpolation', as in "eve/test/{{i}}". this inherits implicit string conversion from JS, but
i think there is an ongoing discussion about how much implicit string conversion makes sense